### ISSUE TITLE
[0.2] Print the error message when a non `2xx` response is returned

### ISSUE THREAD

**Issue by tasn** [svix/svix-cli#13]
Run
```
svix application create --data-name "Test Application" --data-uid "test"
# And again
svix application create --data-name "Test Application" --data-uid "test"
```
The second one will fail with a 409. We should print the json response (pretty-print). Now we just print nothing.

**Comment by svix-frank** [comment]
> Run
> 
> ```
> svix application create --data-name "Test Application" --data-uid "test"
> # And again
> svix application create --data-name "Test Application" --data-uid "test"
> ```
> 
Would we want to print this to stdout or stderr?

**Comment by tasn** [comment]
Good question... Probably for now print it to stderr (same as error) as it's informative.. In the future we can have a flag to also pipe errors to stdout or something for.scripting.

**Comment by tasn** [comment]
Actually.... Maybe stdout!

**Comment by svix-frank** [comment]
closed in #30
